explorer/app/project/api/service.ts

```diff
@@ -22,36 +22,40 @@ const DEFAULT_LIST_PARAMS: ListParams = {
 
 /**
  * Request to get a list of projects.
+ * @param apiPath, Path that will be used to compose the API url
  * @param listParams Params to be used on the request. If none passed, it will default to page's size 25 and the current catalog version
- * @returns @see ProjectListResponse
+ * @returns @see ListResponseType
  */
 export const list = async (
+  apiPath: string,
   listParams?: ListParams
-): Promise<ProjectListResponse> => {
+): Promise<ListResponseType> => {
   const params = { ...DEFAULT_LIST_PARAMS, ...listParams };
-  const res = await fetch(`${URL}index/projects?${convertUrlParams(params)}`);
+  const res = await fetch(`${URL}${apiPath}?${convertUrlParams(params)}`);
   return await res.json();
 };
 
 /**
  * Recursivally call the endpoint to get a list of project. This will iterate over the project list until the next property comes null
```

Comment:
```suggestion
 * Recursively call the endpoint to get a list of project. This will iterate over the project list until the next property comes null
```